hack/observability/cluster-api-visualizer/fetch-visualizer-resources.sh

```diff
@@ -18,12 +18,12 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-CHART_RELEASE=${CHART_RELEASE:-0.1.0}
+CHART_RELEASE=${CHART_RELEASE:-1.0.0}
```

Comment:
are you planning on cutting a github v1.0.0 release along with this chart release https://github.com/Jont828/cluster-api-visualizer? 

That would be a cool announcement to make in CAPI channels

Reply:
I haven't really thought about that at the moment but I can look into it!